moped-editor/src/components/GridTable/Filters.js

```diff
@@ -8,16 +7,14 @@ import {
   FormControl,
   Grid,
   Hidden,
-  Icon,
   IconButton,
   Grow,
   Typography,
 } from "@mui/material";
 import RadioGroup from "@mui/material/RadioGroup";
 import Radio from "@mui/material/Radio";
 import FormControlLabel from "@mui/material/FormControlLabel";
-
-import makeStyles from "@mui/styles/makeStyles";
+import { DeleteOutline, Close, Search, PlaylistAdd } from "@mui/icons-material";
```

Comment:
for some reason the `sx` prop does not seem to work on font icons so I replaced all of these icons imported directly from MUI. see https://stackoverflow.com/questions/74068619/react-change-size-of-icon-received-as-props